bot/cogs/match.py

```diff
@@ -92,6 +104,10 @@ def _pick_player(self, picker, pickee):
             raise PickError(f'Team {picker.mention} is full')
 
         if not picker == pickee:
+            if picker == self.captain_one:
+                self.current_picker = 'team2'
+            else:
+                self.current_picker = 'team1'
```

Comment:
Can this be done with a ternary operator?

Something along the lines of:
```
self.current_picker = picker == self.captain_one ? 'team2' : 'team1'
```

Reply:
`self.current_picker = 'team1' if picker is self.captain_one else 'team2'`

Reply:
Oh I reversed it :laughing: 
` self.current_picker = 'team2' if picker is self.captain_one else 'team1'`

Reply:
@cameronshinn done #18